src/proj.h

```diff
@@ -803,6 +803,8 @@ typedef enum
     PJ_TYPE_TEMPORAL_DATUM,
     PJ_TYPE_ENGINEERING_DATUM,
     PJ_TYPE_PARAMETRIC_DATUM,
+
+    PJ_TYPE_DERIVED_PROJECTED_CRS,
```

Comment:
I added it at the end of the enum for backward compatibility. If it is better  anywhere else, let me know.